extensions/amp-analytics/integrating-analytics.md

```diff
@@ -0,0 +1,58 @@
+# Integrate your analytics tools in AMP HTML
+
+## Overview
+
+If you operate a software-as-a-service tool for publishers to better understand their traffic and visitors, you may want to integrate your service into AMP-Analytics. This will enable your customers to view traffic patterns in AMP HTML.
+
+## Before you begin
+
+Before you can add your analytics service to AMP HTML runtime, you may need to:
+* Identify the kinds of [variables](analytics-vars.md) and [requests](amp-analytics.md#requests) you'll need in an AMP HTML document for your analytics service.
+* Identify the triggers that result in analytics requests being sent from a page that would be relevant for your service.
+* Consider if and how you will [track users across](cross-origin-tracking.md) first-party and third-party AMP contexts.
+* Determine how your analytics dashboard handles AMP traffic.
+* Identify any missing functionality in AMP-Analytics, and note them as requested features on the GitHub project.
+* AMP Analytics sends its variables as query string parameters in a HTTPS request to a preconfigured endpoint.  If you do not already have an existing endpoint, review [this sample](https://github.com/ampproject/amp-publisher-sample#amp-analytics-sample) for an overview on how to build one.  
+* Consider how integration with AMP-Analytics may impact any policies (particularly your privacy policy) or agreements you may have.
+
+## Adding your configuration to the AMP HTML runtime
+
+1. Create an [Intent-To-Implement issue]() stating that you'll be adding your analytics service's configuration to AMP HTML's runtime.
+2. Develop a patch that implements the following:
+ * A new block in [vendors.js](0.1/vendors.js) including any options above and beyond the default, such as:
+  * "vars": {} for additional default variables.
+  * "requests": {} for requests that your service will use.
+  * "optout": if needed
```

Comment:
See #1844. Suggest adding something here that says to raise opt-out in your intent to implement as a design consideration in order to work with project leads on a solution.